### DISCUSSION TITLE
Attribute listing attribute_group is not readable

### DISCUSSION THREAD

**Discussion by zssvaidar** [atrocore/atrocore#811] [Q&A] [answered]
![image](https://github.com/atrocore/atrocore/assets/30953478/1c207931-77b3-4497-b40a-13a9db7fda1e)
After enabling attribute group in list of Attribute page, attribute_group shows it's id not label

**Comment by rratsun** [comment, marked as answer]
It can be if Attribute Group has no name, or it has already deleted.
